src/main/java/kitchenpos/order/application/OrderValidatorImpl.java

```diff
@@ -0,0 +1,36 @@
+package kitchenpos.order.application;
+
+import static kitchenpos.order.domain.OrderStatus.COOKING;
+import static kitchenpos.order.domain.OrderStatus.MEAL;
+
+import java.util.Arrays;
+import java.util.List;
+import kitchenpos.order.dao.OrderDao;
+import kitchenpos.table.application.OrderValidator;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderValidatorImpl implements OrderValidator {
+
+    private static final List<String> CRITERIA_ORDER_STATUS = Arrays.asList(COOKING.name(), MEAL.name());
+
+    private final OrderDao orderDao;
+
+    public OrderValidatorImpl(final OrderDao orderDao) {
+        this.orderDao = orderDao;
+    }
+
+    @Override
+    public void ValidateOrderTableId(final Long orderTableId) {
```

Comment:
```suggestion
    public void validateOrderTableId(final Long orderTableId) {
```
소소하지만 메소드명에 첫문자가 오타가 난 듯하네요 :)